src/src/com/microsoft/aad/adal/HashMapExtensions.java

```diff
@@ -95,17 +96,13 @@
      * @throws JSONException
      */
     static final HashMap<String, String> getJsonResponse(HttpWebResponse webResponse) throws JSONException{
-        HashMap<String, String> response = new HashMap<String, String>();
-        if(webResponse != null && webResponse.getBody() != null && webResponse.getBody().length != 0){
-            JSONObject jsonObject = new JSONObject(
-                    new String(webResponse.getBody()));
-
+        HashMap<String, String> response = new HashMap<>();
```

Comment:
I know you don't touch this, but I'm just wondering why not returning the generic Map.